gmail/src/models/email.ts

```diff
@@ -68,4 +70,29 @@ export class Email {
 
         return email;
     }
+
+    /**
+     * Return the list of the attachments in the email.
+     * Done in a getter and not as a property because this object is serialized and
+     * given to the event handler.
+     */
+    getAttachments(): Attachment[] {
+        const message = GmailApp.getMessageById(this.messageId);
+        const gmailAttachments = message.getAttachments();
+        const attachments: Attachment[] = [];
+
+        // 10 MB maximum per file
```

Comment:
What is enforcing this 10MB limit? Gmail? Odoo?

Reply:
To be sure to not exceed the Gmail quota (our HTTP post request can have 50MB maximum)

Reply:
See https://developers.google.com/apps-script/guides/services/quotas#current_limitations

Reply:
But most of the time with modern mail client, the attachments are automatically added in Gdrive / Onedrive when a limit is exceed (25 for gmail, 33mb for onedrive)

![screenshot_spect](https://user-images.githubusercontent.com/54316226/121638387-2d0ca380-ca8b-11eb-8e8d-2cb3a800d92a.png)